mmcv/ops/csrc/parrots/iou3d.cpp

```diff
@@ -189,7 +189,7 @@ void iou3d_nms_normal_forward(Tensor boxes, Tensor keep, Tensor keep_num,
     int64_t *keep_data = keep.data_ptr<int64_t>();
     int64_t *keep_num_data = keep_num.data_ptr<int64_t>();
 
-    const int col_blocks = DIVUP(boxes_num, THREADS_PER_BLOCK_NMS);
+    const int col_blocks = GET_BLOCKS(boxes_num, THREADS_PER_BLOCK_NMS);
```

Comment:
what if actual `col_blocks` is greater than 4096?

Reply:
As above